packages/hint-validate-summary/src/hint.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @fileoverview check for summary tag and look for the computedStyle to check whether it is the same as it should be
+ */
+
+import { HintContext } from 'hint/dist/src/lib/hint-context';
+import { IHint } from 'hint/dist/src/lib/types';
+import { StyleEvents, StyleParse } from '@hint/parser-css';
+import meta from './meta';
+import selectorParser = require('postcss-selector-parser');
+
+export default class ValidateSummaryHint implements IHint {
+    public static readonly meta = meta;
+    public readonly requiredAttribute = 'display';
+    public readonly requiredValue = 'list-item';
+    public parser = selectorParser();
```

Comment:
Other than `meta` none of these should be `public` as they're for internal use. I'd recommend also making them `static` or moving their declarations outside the class (e.g. `const parser = selectorParser();`) as they don't need to be re-declared per-instance.